services/dropbox.yml

```diff
@@ -0,0 +1,79 @@
+id: dropbox
+name: Dropbox
+rules:
+  - '||addtodropbox.com^'
+  - '||app.hellosign.com^'
+  - '||dash.ai^'
+  - '||db.tt^'
+  - '||docsend.com^'
+  - '||dropbox.com^'
+  - '||dropboxapi.com^'
+  - '||dropboxbusiness.com^'
+  - '||dropboxcaptcha.com^'
+  - '||dropboxforums.com^'
+  - '||dropboxforum.com^'
+  - '||dropboxinsiders.com^'
+  - '||dropboxlegal.com^'
+  - '||dropboxmail.com^'
+  - '||dropboxpartners.com^'
+  - '||dropboxstatic.com^'
+  - '||dropboxteam.com^'
+  - '||dropbox.tech^'
+  - '||dropbox.zendesk.com^'
+  - '||getdropbox.com^'
+  - '||learn.dropbox.com^'
```

Comment:
Subdomains are not needed, please remove them

Reply:
`dropbox.zendesk.com` should be with a subdomain

Reply:
Maybe. Not a problem, if it will be blocked if whole Dropbox blocked.